cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -879,16 +879,7 @@ def equivalent_gate_before(self, after: 'SingleQubitCliffordGate') -> 'SingleQub
         return self.merged_with(after).merged_with(self**-1)
 
     def __repr__(self) -> str:
-        x = self.pauli_tuple(pauli_gates.X)
-        y = self.pauli_tuple(pauli_gates.Y)
-        z = self.pauli_tuple(pauli_gates.Z)
-        x_sign = '-' if x[1] else '+'
-        y_sign = '-' if y[1] else '+'
-        z_sign = '-' if z[1] else '+'
-        return (
-            f'cirq.SingleQubitCliffordGate(X:{x_sign}{x[0]!s}, '
-            f'Y:{y_sign}{y[0]!s}, Z:{z_sign}{z[0]!s})'
-        )
+        return f'cirq.SingleQubitCliffordGate(_clifford_tableau={self._clifford_tableau!r})'
```

Comment:
Using a "private" argument in a repr that cirq produces sends mixed signals: maybe you shouldn't use it in your own code because it's private, but then again cirq core itself depends on it so maybe you can use it after all?

Maybe it's time we remove the underscore? (Deprecating the argument with underscore.)

Reply:
LGTM!

Reply:
Done.